spec/models/post_spec.rb

```diff
@@ -176,28 +203,6 @@
       create(:reply, post: post, user: post.user)
       expect(post.edited_at).to eq(post.created_at)
     end
-
-    it "should update correctly when characters are edited" do
```

Comment:
Is this stuff duplicated in `reply_spec.rb`, then?

Reply:
& stuff below

Reply:
`#has_icons` was duplicated; we don't have the entire `#edited_at` thing on replies so that isn't duplicated